test/express-tests.test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var path = require('path'),
+  helpers = require('yeoman-generator').test,
+  assert = require('yeoman-generator').assert,
+  temp = require('temp').track();
+
+describe('Express Sub Generators Tests', function () {
+  this.timeout(0);
+  /**
+   * Setup the temp directory
+   */
+  before(function (done) {
+    helpers.testDirectory(path.join(__dirname, 'temp'), done);
+  });
+
+  /**
+   * Clean up temp directory
+   */
+  after(function () {
+    temp.cleanup();
+  });
+
+  describe('Generate an express tests through the sub-generator', function () {
+    beforeEach(function (done) {
+      helpers.run(path.join(__dirname, '../express-tests'))
+        .withOptions({
+          'skip-install': true
+        })
+        .withArguments(['foo'])
```

Comment:
That's the name of the test+model. One module can contain several models => several tests.